demo/demo.md

```diff
@@ -24,6 +16,73 @@ The default component supports the basic input `type="text"` structure. The `(op
   }
 </style>
 
+# Input
+
+Use the `auro-input` custom element to create basic single-line text fields. Supports type `text`, `password`, and `email` with validation, required input, error states and a secondary `bordered` theme. Use the slots `label` and `helptext` for additional content support.
+
+## Hacking on name autocomplete
+
+This example shows that you can define the `autocomplete` scenario with `name="name"`. The second example uses `name="name" autocomplete="off"`.
+
+<div class="exampleWrapper exampleWrapper--flex">
+  <auro-input id="name" bordered name="name">
+    <span slot="label">First Name</span>
+    <span slot="helptext">Please enter your full name.</span>
+  </auro-input>
+  <auro-input id="name" bordered name="name" autocomplete="off" required>
+    <span slot="label">Last Name</span>
+    <span slot="helptext">Please enter your full name.</span>
+  </auro-input>
+</div>
+
+## Hacking on email autocomplete
+
+<div class="exampleWrapper exampleWrapper--flex">
+  <auro-input id="email-address" bordered name="email">
+    <span slot="label">Email address</span>
+    <span slot="helptext">Please enter your email address.</span>
+  </auro-input>
+  <auro-input id="email-address" bordered name="email" autocomplete="off" required>
+    <span slot="label">Email address</span>
+    <span slot="helptext">Please enter your email address.</span>
+  </auro-input>
+</div>
+
+## Hacking on street address autocomplete
+
+<div class="exampleWrapper exampleWrapper--flex">
+  <auro-input id="street-address" bordered name="address">
```

Comment:
Shouldn't we use some [autocomplete](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) values here? I'm not sure the name is enough. e.g. autocomplete="address-line1"

Also, it might be helpful to put a regular `<input>` here for comparison, so we can determine whether something is broken for just shadow DOM or for regular inputs as well.

Reply:
This is what I am learning, HTML spec be damned... none of the browsers are consistent and support is inconsistent. The only thing that is consistent is `autocomplete="off"`

I can update the demo to include a native input, but from initial testing what I am discovering is that `autocomplete` does not work within the scope of the shadow DOM other than `off`. 

This just leads me more to the fact that we can release this with a disclaimer and limited support.